docs/content/ingestion/index.md

```diff
@@ -91,6 +91,54 @@ If `type` is not included, the parser defaults to `string`.
 | type | String | This should say `protobuf`. | no |
 | parseSpec | JSON Object | Specifies the format of the data. | yes |
 
+### Avro Stream Parser
+
+The `timeAndDims` parseSpec is good to use with avro hadoop parser.
+ 
+| Field | Type | Description | Required |
+|-------|------|-------------|----------|
+| type | String | This should say `avro_stream`. | no |
+| avroBytesDecoder | JSON Object | Specifies the how to decode bytes to Avro record. | yes |
+| parseSpec | JSON Object | Specifies the format of the data. | yes |
+
+#### Avro Bytes Decoder
+
+If `type` is not included, the avro bytes decoder defaults to `schema_repo`.
+
+##### SchemaRepo Based Avro Bytes Decoder
+
+This Avro bytes decoder first extract `subject` and `id` from input message bytes, then use them to lookup the Avro schema with which to decode Avro record from message payload. Details can be found in [schema repo](https://github.com/schema-repo/schema-repo) and [AVRO-1124](https://issues.apache.org/jira/browse/AVRO-1124). You will need an http service like schema repo to hold the avro schema. Towards the schema registration code in message producer side, you can refer to `io.druid.data.input.AvroStreamInputRowParserTest#testParse()`.
+
+| Field | Type | Description | Required |
+|-------|------|-------------|----------|
+| type | String | This should say `schema_repo`. | no |
+| subjectAndIdConverter | JSON Object | Specifies the how to extract subject and id from message bytes. | yes |
+| schemaRepository | JSON Object | Specifies the how to lookup Avro schema from subject and id. | yes |
+
+##### Avro-1124 Subject And Id Converter
+
+| Field | Type | Description | Required |
+|-------|------|-------------|----------|
+| type | String | This should say `avro_1124`. | no |
+| topic | String | Specifies the topic of your kafka stream. | yes |
+
+
+##### Avro-1124 Schema Repository
+
+| Field | Type | Description | Required |
+|-------|------|-------------|----------|
+| type | String | This should say `avro_1124_rest_client`. | no |
+| url | String | Specifies the endpoint url of your Avro-1124 schema repository. | yes |
+
+### Avro Hadoop Parser
+
+The `inputFormat` of `inputSpec` in `ioConfig` must be set to "io.druid.data.input.avro.AvroValueInputFormat". You may want to set Avro reader schema in `jobProperties` in `tuningConfig`, eg: `"avro.schema.path.input.value": "/path/to/your/schema.avsc"`, or `"avro.schema.input.value": "your_schema_JSON_object"`. The `timeAndDims` parseSpec is good to use with avro hadoop parser.
+
+| Field | Type | Description | Required |
+|-------|------|-------------|----------|
+| type | String | This should say `avro_hadoop`. | no |
+| parseSpec | JSON Object | Specifies the format of the data. | yes |
+
```

Comment:
nit: it will be nice to have actual json example one for the hadoop parser and one for the streaming parser.